### DISCUSSION TITLE
Trying to convert one JSON format to another. Copilot decided to do something really unexpected

### DISCUSSION THREAD

**Discussion by Xerios** [community/community#46522] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Looks like copilot poorly googled an answer, and had a stroke while trying to copy some forum's content.

Images speak for themselves.

![image](https://user-images.githubusercontent.com/181384/217018078-1f9f02b3-d558-45f6-a96d-a98dbc9e63d3.png)
![image](https://user-images.githubusercontent.com/181384/217018136-6e1ca110-08e8-4398-b45a-fff25844630a.png)
![image](https://user-images.githubusercontent.com/181384/217018227-f7d8014e-1def-4746-8062-4335b3652163.png)

The way I triggered this was by synthesizing solutions in a newly created file (unsaved), and the code was something like:
````js
const test = { OLD_JSON }

/**
Transform to
{ NEW_JSON }
*/```

**Comment by vincolus** [comment]
having the same issues. currently, it is more a distraction than of any help.
it looked way better in advertisement than it is in real-life.
